mne/forward/_compute_forward.py

```diff
@@ -126,22 +209,41 @@ def _bem_specify_coils(bem, coils, coord_frame, mults, n_jobs):
 
 
 def _bem_specify_els(bem, els, mults):
-    """Set up for computing the solution at a set of electrodes"""
+    """Set up for computing the solution at a set of EEG electrodes.
+
+    Parameters
+    ----------
+    bem : dict
+        BEM information
+    els : list of dict, len(n_EEG_sensors)
+        List of EEG sensor information dicts
+    mults: ndarray, shape (1, n_BEM_vertices)
```

Comment:
is this actually 2D? I think it's 1D which would be `shape (n_BEM_vertices,)`. this shows up a few places, so if it's wrong here please fix everywhere